internal/repository/cluster_config/mongo/delete_capsule_config.go

```diff
@@ -0,0 +1,26 @@
+package mongo
+
+import (
+	"context"
+
+	"github.com/rigdev/rig/pkg/auth"
+	"github.com/rigdev/rig/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func (c *MongoRepository) DeleteCapsuleConfig(ctx context.Context, capsuleName string) error {
```

Comment:
Vi skal nok lige søge efter `capsuleName` i kode basen. Bare så vi er konsistente med om det er ID eller Name.

```suggestion
func (c *MongoRepository) DeleteCapsuleConfig(ctx context.Context, capsuleID string) error {
```

Reply:
Done!